AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/TestMatrixStore.kt

```diff
@@ -140,6 +141,63 @@ internal class TestMatrixStore(
         return newTestMatrix
     }
 
+    /**
+     * Creates a [TestMatrix] to run the tests specified in [testTargets] list
+     * using the same configuration as [testMatrix]
+     */
+    suspend fun createTestMatrix(
+        testMatrix: TestMatrix,
+        testTargets: List<String>? = null
```

Comment:
Yeah, I had kept it nullable since in the base testMatrix ftl model, it's nullable. But, I don't see any issue in making in non-null and non-empty. I'll do that.

Reply:
Done